apps/css-workshop/src/components/Checkbox.astro

```diff
@@ -40,19 +39,18 @@ const { status, isLoading, indeterminate, disabled, variant, basic, ...props } =
           [`iui-${status}`]: !!status,
         },
       ]}
-      {...props}
     >
       <input
```

Comment:
I think you can store this `input` in a variable in the frontmatter, since it is repeated twice